src/net_processing.h

```diff
@@ -116,8 +116,13 @@ class PeerManager : public CValidationInterface, public NetEventsInterface
     /** Get peer manager info. */
     virtual PeerManagerInfo GetInfo() const = 0;
 
-    /** Relay transaction to all peers. */
-    virtual void RelayTransaction(const Txid& txid, const Wtxid& wtxid) = 0;
+    /**
+     * Schedule a transaction to be broadcast to all peers at a later time.
```

Comment:
nit: "at a later time" is implied by "schedule", unnecessary to mention, plus it may suggest to the reader that you can control the later time here (which is not the case).

```suggestion
     * Schedule a transaction for broadcasting to all peers.
```